src/api/notification/content-types/notification/schema.json

```diff
@@ -0,0 +1,36 @@
+{
+  "kind": "collectionType",
+  "collectionName": "notifications",
+  "info": {
+    "singularName": "notification",
+    "pluralName": "notifications",
+    "displayName": "Notification",
+    "description": ""
+  },
+  "options": {
+    "draftAndPublish": false
+  },
+  "pluginOptions": {},
+  "attributes": {
+    "Title": {
+      "type": "string"
+    },
+    "idea_card": {
```

Comment:
Would we need to create a field for each new collection that wants notifications? Maybe we can create an enum field for the collection type (idea-card, comment, etc) and another field for ID of the object that triggered the notification.
The downside is we don't have a relationship at the database level, so we can't do things like cascade delete directly from database.

Reply:
@chungthuang Just made the changes to the notification collection schema to reflect these